src/wallet/wallet.cpp

```diff
@@ -545,7 +545,6 @@ void CWallet::SetMinVersion(enum WalletFeature nVersion, WalletBatch* batch_in)
             delete batch;
     }
 }
-
```

Comment:
In bf9deb407a8d73ac3b645b34c9bd6d28729a809c "rpc/wallet: Add details and duplicate section for simulaterawtransaction"

nit: Unrelated whitespace change